Assets/Scripts/Flying.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;

```

Comment:
Need to add our copyright to the top
```suggestion
// Copyright (c) TigardHighGDC
// SPDX-License SPDX-License-Identifier: Apache-2.0

using System.Collections;
```